README.md

```diff
@@ -222,58 +220,191 @@ example of an 'edgy' configuration.
-
-#### `OGPTActAs`
+### `OGPT`
+`OGPT` command opens an interactive window to communicate with LLM backends. The interactive window consists of four panes:
+* OGPT Chat: Input text area to write prompts.
+* OGPT: Output text area printing responses from LLM.
+* OGPT Parameters: (Toggled by \<Ctrl-o\>,) Parameter panel to display and modify LLM paramters.
+* OGPT Sessions: (Toggled by \<Ctrl-o\>,) Sessions panel to display chat sessions.
+
+#### Useful shortcuts
+<table>
+    <thead>
+        <tr>
+            <th>Area</th>
+            <th>Default shortcuts</th>
+            <th>Description</th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td rowspan=6>Common</td>
+            <td>Ctrl-o</td>
+            <td>Toggle parameter panel (OGPT Parameters) and sessions panel (OGPT Sessions).</td>
+        </tr>
+        <tr>
+            <td>Ctrl-n</td>
+            <td>Create a new session.</td>
+        </tr>
+        <tr>
+            <td>Ctrl-c</td>
+            <td>Close OGPT.
+        </tr>
+        <tr>
+            <td>Ctrl-i</td>
+            <td>Copy code in LLM's latest response in OGPT output text area.
+        </tr>
+        <tr>
+            <td>Ctrl-x</td>
+            <td>Stop generating response.</td>
+        </tr>
+        <tr>
+            <td>Tab</td>
+            <td>Cycle through panes.</td>
+        </tr>
+        <tr>
+            <td rowspan=4>OGPT</td>
+            <td>K</td>
+            <td>Previous response.</td>
+        </tr>
+        <tr>
+            <td>J</td>
+            <td>Next response.</td>
+        </tr>
+        <tr>
+            <td>Ctrl-u</td>
+            <td>Scroll up.</td>
+        </tr>
+        <tr>
+            <td>Ctrl-d</td>
+            <td>Scroll down.</td>
+        </tr>
+        <tr>
+            <td rowspan=5>OGPT Chat</td>
+            <td>Enter<br>(normal mode)</td>
+            <td>Send prompt to LLM.</td>
+        </tr>
+        <tr>
+          <td>Alt-Enter<br>(input mode)</td>
+          <td>Send prompt to LLM.</td>
+        </tr>
+        <tr>
+          <td>Ctrl-y</td>
+          <td>Copy the latest response from LLM in OGPT output text area.</td>
+        </tr>
+        <tr>
+          <td>Ctrl-r</td>
+          <td>Toggle role (assistant or user).</td>
+        </tr>
+        <tr>
+          <td>Ctrl-s</td>
+          <td>Toggle system message.</td>
+        </tr>
+        <tr>
+          <td rowspan=1>OGPT Parameters</td>
+          <td>Enter</td>
+          <td>Change parameter.</td>
+        </tr>
+        <tr>
+            <td rowspan=3>OGPT Sessions</td>
+            <td>Enter</td>
+            <td>Switch session.</td>
+        </tr>
+        <tr>
+            <td>d</td>
+            <td>Delete session.</td>
+        </tr>
+        <tr>
+            <td>r</td>
+            <td>Rename session. Note that an active session cannot be removed.</td>
+        </tr>
+    </tbody>
+</table>
+
+<!-- Someone please fix the ugly HTML table to Markdown table -->
```

Comment:
I wished this markdown table format to work beautifully with merged rows by using `^`, but it failed badly. Still, I'm leaving this table as a comment in case someone in the future knows how to fix this beautifully.